doc/classes/TreeItem.xml

```diff
@@ -96,6 +96,14 @@
 				Returns the button index if there is a button with ID [param id] in column [param column], otherwise returns -1.
 			</description>
 		</method>
+		<method name="get_button_color" qualifiers="const">
+			<return type="Color" />
+			<param index="0" name="column" type="int" />
+			<param index="1" name="id" type="int" />
+			<description>
+				Returns the color of the button with ID [param id] in column [param column]. If the specified button does not exist, returns [code skip-lint]Color()[/code].
```

Comment:
Almost there! We agreed to refer to the constant in the description.
```suggestion
				Returns the color of the button with ID [param id] in column [param column]. If the specified button does not exist, returns [constant Color.BLACK].
```

Reply:
The problem with doing that is that `Color()` doesn't always return the equivalent of `Color.BLACK`. In the C#, `Color()` instead returns the equivalent of `Color(0, 0, 0, 0)`.

So what should I do?

Reply:
That's *even more* of a reason to use the constant to refer to it. The source code of the engine will not behave any differently, no matter what scripting language you use.

In C++ the color constructor returns `Color(0, 0, 0, 1)`, which corresponds to **Color.BLACK**. If you call this function in C# and it fails, the returned value is still `Color(0, 0, 0, 1)`.